BlueprintUI/Sources/Element/EnvironmentAdaptingStorage.swift

```diff
@@ -10,12 +13,19 @@ struct EnvironmentAdaptingStorage: ContentStorage {
     var adapter: (inout Environment) -> Void
 
     var child: Element
+    var content: ElementContent
```

Comment:
I'm not 100% sure we can just grab this and cache it all the time, eg some elements, like `GeometryReader`, can change the end result of the content based on the size, or others, like `EnvironmentReader`, based on `Environment`. In the caching branch [I'd introduced `isDynamicElement`](https://github.com/square/Blueprint/pull/418/files#diff-07c71c5dcdc5da4c2ddf51b560695e562af396d4afafe3908b35656a0907459aR42) (which would then disable caching of `content`) to fix this.